pype/plugins/maya/publish/collect_review.py

```diff
@@ -69,7 +69,12 @@ def process(self, instance):
                     instance.data['remove'] = True
                 i += 1
         else:
-            instance.data['subset'] = task + 'Review'
+            subset = "{}{}{}".format(
```

Comment:
Works well, however we'll need to add backwards compatible option in here. Some of our clients are relying on this in production now and have custom scripts and setup locked to previous neming (without the subset name), we can't change this mid-project. 

Let's please add a switch configurable using `self.legacy=true` presetable attribute, that defaults to True and when on behaves as before. Like that it can go out in 2.12 at the end of the week. 

From 2.13 or 14, we can then switch it OFF by default. and in pype 3 remove it. what do you think?

Reply:
Sounds like a good idea. Done.